web/templates/layout/volunteer_app_navbar.html.eex

```diff
@@ -0,0 +1,8 @@
+<%= CoursePlanner.SharedView.navbar "Course Planner" do %>
+  <%= CoursePlanner.SharedView.navbar_item "Dashboard", @conn, dashboard_path(@conn, :show) %>
+  <%= CoursePlanner.SharedView.navbar_item "Calendar", @conn, schedule_path(@conn, :show) %>
+  <%= CoursePlanner.SharedView.navbar_separator %>
+  <%= CoursePlanner.SharedView.navbar_item "Tasks", @conn, task_path(@conn, :index) %>
+  <%= CoursePlanner.SharedView.navbar_separator %>
```

Comment:
yeah
```eex
<%= if show_program_about?() do %>
  <%= CoursePlanner.SharedView.navbar_separator %>
  <%= CoursePlanner.SharedView.navbar_item "About #{get_program_name()}", @conn, about_path(@conn, :show) %>
<% end %>
```
instead of the current